lib/WorkflowMain.groovy

```diff
@@ -71,12 +71,6 @@ class WorkflowMain {
 
         // Check AWS batch settings
         NfcoreTemplate.awsBatch(workflow, params)
-
-        // Check input has been provided
-        if (!params.input) {
-            log.error "Please provide an input samplesheet to the pipeline e.g. '--input samplesheet.csv'"
-            System.exit(1)
-        }
```

Comment:
I would not remove that

Reply:
we need to have some work around for utilizing the automatic restart. This check totally negates the checks at `workflows/sarek` lines 46 - 56

Reply:
hm ok I would say we need to fix sarek not the template then, what do you think?

Reply:
I think it's quite a useful feature to be able to restart the analysis. There is already a warning, and if the step isn't there it throws an error. You need to designate a step, which essentially is a shortcut for editing the config file. just set step = `variant_calling` instead of `input = 'Path/To/Dir/preprocessing/csv/recalibrated.csv'`

Reply:
right, I definitly think we should keep it, just make it work properly without messing with the template :)

Reply:
or otherwise we should raise the issue in tools and see i the template needs changing